pom.xml

```diff
@@ -31,6 +31,34 @@
     <build>
```

Comment:
- [ ] What is the "Add .factorypath to .gitignore" in  [Possible Solution](https://github.com/terasolunaorg/terasoluna-gfw-web-blank/issues/525#issue-1367492349)?

Reply:
6ca0dfca663ea25afba9a0a66ff5ca477b59322a
`.factorypath` is the file generated when compiled using MapStruct.
It only contains information such as the version of mapstruct-processor used at the time of compilation, and is unnecessary for the project.
So I added it to `.gitignore`.